models/iaf_psc_alpha_multisynapse.nestml

```diff
@@ -47,9 +47,8 @@ neuron iaf_psc_alpha_multisynapse_neuron:
     tau_syn ms[receptors] = 2ms # Time constant of synaptic current.
     t_ref   ms = 2ms            # Refractory period.
     E_L     mV = -70mV          # Resting potential.
-    function V_reset mV = -70.0mV - E_L # Reset value of the membrane potential
-    function V_th   mV = -55.0mV - E_L # Threshold, RELATIVE TO RESTING POTENTIAL(!).
-                                    # I.e. the real threshold is (E_L+V_th).
+    V_reset mV = 0mV            # Reset value of the membrane potential
```

Comment:
I did not notice this before, but you should not mix absolute (e.g. ``E_L``) and relative values here. Since these are example models, they should be clear to the user: use absolute values for all parameters.
If you want to work with relative values in the code, then declare aliases and use them in the ODEs ;)